### PR TITLE
Revise sdw use no pm patches

### PR DESCRIPTION
**Pull request by bardliao**
Address commits from maintainers. 
1. Remove fixed tags
2. Separate commits
No code changed.

### CODE REVIEW

File: drivers/base/regmap/regmap-sdw-mbq.c [hunk 1]
```diff
@@ -98,4 +98,4 @@ struct regmap *__devm_regmap_init_sdw_mbq(struct sdw_slave *sdw,
 EXPORT_SYMBOL_GPL(__devm_regmap_init_sdw_mbq);
 
 MODULE_DESCRIPTION("Regmap SoundWire MBQ Module");
-MODULE_LICENSE("GPL v2");
+MODULE_LICENSE("GPL");
```
Comment (plbossart) [inline drivers/base/regmap/regmap-sdw-mbq.c]: the commit message could be misleading. We are not changing the license at all.

use MODULE_LICENSE("GPL")

also add a pointer to the licensing rules for reference.

**Review by plbossart — CHANGES_REQUESTED** [review]
last patch commit subject is misleading.

Also check with Vinod Koul if he really meant removing all the fixes tag, we want these changes back-ported to 5.10.

**Comment by bardliao** [comment]
> last patch commit subject is misleading.
> 
> Also check with Vinod Koul if he really meant removing all the fixes tag, we want these changes back-ported to 5.10.

I saw you have replied Vinod, so I will wait for his reply. BTW, we can't partially back-port these changes, right?

**Comment by plbossart** [comment]
> > last patch commit subject is misleading.
> > Also check with Vinod Koul if he really meant removing all the fixes tag, we want these changes back-ported to 5.10.
> 
> I saw you have replied Vinod, so I will wait for his reply. BTW, we can't partially back-port these changes, right?

I don't know what you mean by 'partially back-port these changes'? We need all of these, if you cherry-pick then 5.10 w/ some but not all fixes will not work at all.

**Comment by bardliao** [comment]
> > > last patch commit subject is misleading.
> > > Also check with Vinod Koul if he really meant removing all the fixes tag, we want these changes back-ported to 5.10.
> > 
> > 
> > I saw you have replied Vinod, so I will wait for his reply. BTW, we can't partially back-port these changes, right?
> 
> I don't know what you mean by 'partially back-port these changes'? We need all of these, if you cherry-pick then 5.10 w/ some but not all fixes will not work at all.

Yes, that answered my question. I meant removing some fixes tags and only cherry-picking those with fixes tags to 5.10 will not work.

**Comment by bardliao** [comment]
@plbossart Vinod agree that we put fixes flags. I put them back. And I added below for the ``soundwire: export sdw_write/read_no_pm functions`` commit.
Fixes: 0231453 ('soundwire: bus: add clock stop helpers')
Fixes: 60ee9be ('soundwire: bus: add PM/no-PM versions of read/write functions')
Could you review it?

File: drivers/base/regmap/regmap-sdw-mbq.c (hunk 1 above)
Comment (plbossart) [inline drivers/base/regmap/regmap-sdw-mbq.c]: typo in commit message: "the same".

File: drivers/soundwire/bus.c [hunk 2]
```diff
@@ -478,7 +479,7 @@ int sdw_bwrite_no_pm_unlocked(struct sdw_bus *bus, u16 dev_num, u32 addr, u8 val
 }
 EXPORT_SYMBOL(sdw_bwrite_no_pm_unlocked);
 
-static int
+int
```
Comment (plbossart) [inline drivers/soundwire/bus.c]: single line?

**Review by plbossart — CHANGES_REQUESTED** [review]
two minor cosmetic problems, looks good otherwise.

File: drivers/soundwire/bus.c (hunk 2 above)
Comment (bardliao) [reply drivers/soundwire/bus.c]: updated

File: drivers/base/regmap/regmap-sdw-mbq.c (hunk 1 above)
Comment (bardliao) [reply drivers/base/regmap/regmap-sdw-mbq.c]: updated

**Review by plbossart — APPROVED** [review]
Sounds good @bardliao

**Review by RanderWang — APPROVED** [review]
only one line code change with 16 commit :)

**Pull request merged**